### ISSUE TITLE
4 stars?

### ISSUE THREAD

**Issue by eye-wave** [rubenvar/svelte-star-rating#7]
when given rating a value with more than 2 decimal points
number of stars decreases from 5 to 4

**Comment by rubenvar** [comment]
Hello @Eyewavu, I cannot reproduce this problem. When I use this code:

```svelte
<StarRating
  rating={2.3298767434278}
  config={{
    size: 36,
    showText: true,
  }}
/>
```

I see this result rendered:

![many decimal values](https://user-images.githubusercontent.com/35899750/188644071-99dc1b39-acae-4c2b-a173-6b3a0721e9b1.png)

Could you please elaborate or provide an example or reproduction?

**Comment by eye-wave** [comment]
i saw the problem occurring when using values such as 4.001 3.00002
![Screenshot from 2022-09-08 15-17-09](https://user-images.githubusercontent.com/89079979/189132679-1674b485-35a4-4ba1-9a29-a1d17f44d79a.png)
![Screenshot from 2022-09-08 15-17-32](https://user-images.githubusercontent.com/89079979/189132684-2107809f-14ef-450f-9459-73e5ef13a8f3.png)

**Comment by eye-wave** [comment]
didn't mean to close this issue
damn it missclick

**Comment by rubenvar** [comment]
Right, thank you @Eyewavu, I had not caught this case before.

I have just published version `2.0.1` to npm, which should fix this bug. I will close the issue but please let me know if you still see this problem happening.
